### PR TITLE
[Feat] UI Overhaul

### PR DESCRIPTION
**Pull request by BrettCleary**
UI Overhaul Dev Branch to PR into

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/components/UI/WebviewControls/index.tsx [hunk 1]
```diff
@@ -78,6 +78,12 @@
     [webview]
   )
 
+  if (
+    url.includes('store.hyperplay.xyz') ||
+    url.includes('docs.hyperplaygaming.com')
```
[the remark this answers is not in the document]
Comment (BrettCleary) [reply src/frontend/components/UI/WebviewControls/index.tsx:83]: good bot

**Comment by stormcakes** [comment]
Testing on Windows via a local build and ran into a few issues. 
I see the attached instead of the UI where it should just say MetaMask and have the recovery phrase.
image

When initially loading into the dev or artifact build the user is instantly presented with the MM pop-up - is that intentional? 

When clicking the Discord and Twitter icons I had to press them twice for it to register. After playing around - I noticed that it was more of an issue that the "enabled" field that would trigger the link to open was small. I.e, you are forced to click the actual logos to trigger the action but clicking anywhere in the box that's highlighted doesnt actually work. This is for the Discord and Twitter boxes only. All other boxes seem to work just fine.

**Comment by stormcakes** [comment]
Does this have the most UTD UI from the figma? I'm seeing this in the figma 
image

And this on Local Dev. 
image

**Comment by stormcakes** [comment]
But this might be due to the conflicts above ^^ so not sure.

File: src/frontend/components/UI/WebviewControls/index.tsx [hunk 2]
```diff
@@ -78,6 +78,12 @@
     [webview]
   )
 
+  if (
+    url.includes('https://store.hyperplay.xyz') ||
```
[the remark this answers is not in the document]
Comment (BrettCleary) [reply src/frontend/components/UI/WebviewControls/index.tsx]: fixed

File: src/frontend/components/UI/WebviewControls/index.tsx [hunk 3]
```diff
@@ -78,6 +78,12 @@
...
+    url.includes('https://docs.hyperplaygaming.com')
```
[the remark this answers is not in the document]
Comment (BrettCleary) [reply src/frontend/components/UI/WebviewControls/index.tsx]: fixed

**Comment by BrettCleary** [comment]
> When clicking the Discord and Twitter icons I had to press them twice for it to register. After playing around - I noticed that it was more of an issue that the "enabled" field that would trigger the link to open was small. I.e, you are forced to click the actual logos to trigger the action but clicking anywhere in the box that's highlighted doesnt actually work. This is for the Discord and Twitter boxes only. All other boxes seem to work just fine.

fixed now

**Comment by stormcakes** [comment]
When clicking the drop down to add a wallet or navigate to "manage accounts" on the top right - the "Add Game" overlay remains on top of the list. 
image. This only happens in the Library screen. 

There is a similar issue on the Downloads screen as well. 
image

**Pull request merged**